src/internal_events/add_fields.rs

```diff
@@ -68,6 +68,6 @@ pub struct AddFieldsFieldNotOverwritten<'a> {
 
 impl<'a> InternalEvent for AddFieldsFieldNotOverwritten<'a> {
     fn emit_logs(&self) {
-        error!(message = "Field not overwritten.", %self.field, rate_limit_secs = 30);
+        debug!(message = "Field not overwritten.", %self.field, rate_limit_secs = 30);
```

Comment:
(Same as above but the points are different)

We need to consider 2 possibilities here, each of which is of different importance.

 * The field exists, and we didn't overwrite because we shouldn't have overwritten. This should be debug. (A user would appreciate this information while debugging)
 * The field does not exist, and we did write in. This should be trace (we did take an action which mutated the event)

As I mentioned in #3803 (comment), we might need to pass that variable into this event.